packages/design-tokens/src/tokens/functional/components/video-player/base.json

```diff
@@ -0,0 +1,129 @@
+{
+  "brand": {
+    "VideoPlayer": {
+      "border": {
+        "radius": {
+          "value": "var(--brand-borderRadius-medium)"
+        }
+      },
+      "controls": {
+        "gap": {
+          "value": "15px"
+        },
+        "background": {
+          "value": "rgba(0, 0, 0, 0.2)"
+        },
+        "backgroundGradient": {
+          "value": "linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.6))"
+        },
+        "padding": {
+          "value": "var(--base-size-16) var(--base-size-24)"
+        },
+        "icon": {
+          "height": {
+            "value": "24px"
+          }
+        },
+        "focus": {
+          "borderColor": {
+            "value": "var(--brand-color-focus)"
+          },
```

Comment:
I think you can drop this token and just use `var(--brand-color-focus)` in the stylesheet.

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/373/commits/6f9a9c8740844ac6d50be622f1c9aab106c564dc)